src/chainparams.cpp

```diff
@@ -200,7 +200,7 @@ class CCustomParams : public CChainParams {
 
         genesis = CreateGenesisBlock(consensus, strNetworkID, 1296688602, genesisChallengeScript, 1);
         if (initialFreeCoins != 0) {
-            AppendInitialIssuance(genesis, COutPoint(uint256(commit), 0), parentGenesisBlockHash, 100, initialFreeCoins/100, 0, 0, CScript() << OP_TRUE);
+            AppendInitialIssuance(genesis, COutPoint(uint256(commit), 0), parentGenesisBlockHash, 1, initialFreeCoins, 0, 0, CScript() << OP_TRUE);
```

Comment:
Perhaps we can just remove the parameter and simplify the function. That can be done on a different PR though.

Reply:
Yep thought of that as well, could change this if you want

Reply:
I don't know, might be useful in the future. Meh :)